src/page-modules/assistant/server/journey-planner/journey-gql/via-trip-with-details.gql

```diff
@@ -0,0 +1,127 @@
+query ViaTripsWithDetails(
+  $from: Location!
+  $to: Location!
+  $when: DateTime
+  $via: [ViaLocationInput!]!
+) {
+  viaTrip(
+    from: $from
+    to: $to
+    searchWindow: "PT2H" # 1 hour
```

Comment:
Should maybe be a param instead of a static value?

Reply:
I think it's fine that this is a static value, as this is probably not something we want to change between searches. I see that you write `searchWindow: "PT2H" # 9 hours` in another file, and 1 hour here. Looking at the value it looks to be 2h. Any reason not to use 9 hours as done for regular trips?

Reply:
So it should be "PT9H" to ensure a 9 hours search windows? The "PT2H" value is used for testing to limit the search window, and the comment comes simply from copy pasting from Shamash.

Reply:
We saw that in some cases in Møre and Romsdal there is a while between the buses, yielding no results when searching with a small search window. To fix this we increased the search window to 9 hours, but keeping the number of trip patterns to 8. I don't think this affects the search time, but might be wrong. I see that we can specify `numTripPatterns` in the viaTrip query as well. You could compare the search time when searching with 9 hours and 8 number of trip patterns vs. 2 hours of search window while the number of trip patterns is not specified. 

I can't see any documentation of the `searchWindow` value, but `PT9H` sounds like 9 hours to me. For the trip query, we use 540 (minutes).